src/applications/form-renderer/utils/formConfig.js

```diff
@@ -4,29 +4,51 @@ import {
   dateOfBirthUI,
   fullNameSchema,
   fullNameUI,
+  serviceNumberSchema,
+  serviceNumberUI,
+  ssnOrVaFileNumberSchema,
+  ssnOrVaFileNumberUI,
   titleUI,
 } from 'platform/forms-system/src/js/web-component-patterns';
 import manifest from '../manifest.json';
 import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 
-const selectSchemas = ({ pageTitle, additionalFields }) => {
-  const schemas = {
-    schema: {
+const selectSchemas = ({ pageTitle, type, additionalFields }) => {
+  const schemas = {};
+
+  if (type === 'digital_form_name_and_date_of_bi') {
```

Comment:
Like with the `content-build` PR, this will be converted to a `switch` statement once a third pattern is added.